INSTALL.md

```diff
@@ -94,20 +94,19 @@ permissions (like `system:admin`), as it needs to create a
 `ClusterRole` for deploying function containers from the `fission`
 namespace/project.
 
-Identically as with Kubernetes, you need to set the FISSION_URL and FISSION_ROUTER environment variables. If you're using minishift, use these commands:
-
-```
-  $ export FISSION_URL=http://$(minishift ip):31313¬
-  $ export FISSION_ROUTER=$(minishift ip):31314¬
-```
 #### Using Minishift or Local Cluster
```

Comment:
I see what you mean. One thing I do not have experience with is how relevant the 'using other clouds' section is, since I have only done this on minishift. If you notice, I modified both sections to use https://github.com/fission/fission/releases/download/nightly20170705/fission-cloud.yaml so they are no longer that much different.  It may be that you could use the same oc calls to expose the router and controller in minishift and other clouds.  In any case, it might make sense to separate the FISSION_URL stuff into a separate section since it should work the same in minishift and other clouds.